Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspectorViewModel.js

```diff
@@ -448,26 +478,131 @@ define([
          */
         this.showGeometricError = false;
 
-        var freezeFrame = knockout.observable();
-        knockout.defineProperty(this, 'freezeFrame', {
+        var textureMemory = knockout.observable();
+        knockout.defineProperty(this, 'textureMemory', {
             get : function() {
-                return freezeFrame();
+                return textureMemory();
             },
             set : function(value) {
-                freezeFrame(value);
+                textureMemory(value);
                 if (that._tileset) {
-                    that._tileset.debugFreezeFrame = value;
-                    that._scene.debugShowFrustumPlanes = value;
+                    that._tileset.debugShowTextureMemoryUsage = value;
                 }
             }
         });
         /**
-         * Gets or sets the flag to suspend updates.  This property is observable.
+         * Displays the texture memory used per tile.  This property is observable.
+         * @memberof Cesium3DTilesInspectorViewModel.prototype
          *
          * @type {Boolean}
          * @default false
          */
-        this.freezeFrame = false;
+        this.textureMemory = false;
+
+        var vertexMemory = knockout.observable();
```

Comment:
You didn't miss it! I accidentally reverted it along with the labels-to-view-model commits.